forte_wrapper/vader/sentiment_analysis.py

```diff
@@ -51,24 +51,36 @@ def __init__(self):
 
     def initialize(self, resources: Resources, configs: Config):
         # pylint: disable=unused-argument
+        super().initialize(resources, configs)
         self.sentence_component = configs.get('sentence_component')
 
     def _process(self, input_pack: DataPack):
-        for sentence in input_pack.get(entry_type=Sentence,
-                                       components=self.sentence_component):
-            scores = self.analyzer.polarity_scores(sentence.text)
-            sentence.sentiment = scores
+        path_str, module_str = self.configs.entry_type.rsplit('.', 1)
+
+        mod = importlib.import_module(path_str)
+        entry = getattr(mod, module_str)
+        for entry_specified in input_pack.get(
+                entry_type=entry,
+                components=self.sentence_component):
+            scores = self.analyzer.polarity_scores(entry_specified.text)
+            entry_specified.sentiment = scores
```

Comment:
https://github.com/asyml/forte-wrappers/pull/33/files#diff-48383a329f41770dd084727f0b914b21fafa1a97fe92270517b15d76f194b125R71

Similar to here, let's don't assume the attribute to be `sentiment`. We can pass that as config too.